Custom-Pylint-checks.md

```diff
@@ -527,8 +527,14 @@ def test_finds_hello_world_assignment(self):
 
 Once you've added test cases, you can run them like this:
 
+Python:
 ```console
 python -m scripts.run_backend_tests --test_target=scripts.linters.pylint_extensions_test --verbose
 ```
 
+Docker:
+```console
+make run_backend_tests PYTHON_ARGS="--test_target=scripts.linters.pylint_extensions_test --verbose"
```

Comment:
![Screenshot from 2024-02-08 21-37-29](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/02fe4bba-0320-4ffd-af55-1c73fbf1e5a3)
this error is coming

Reply:
@jaiakash Please fix this error.

Reply:
also corrected the command here also @gp201 but still its throwing some error as shown in image below 
![Screenshot from 2024-02-10 14-51-16](https://github.com/oppia/oppia-web-developer-docs/assets/127002519/49c9b113-f2cc-49cf-8cd3-20546401c7eb)